source/core/src/main/com/csse3200/game/entities/Entity.java

```diff
@@ -64,9 +68,8 @@ public EnemyType getEnemyType() {
   }
 
   // Setter for enemy type
-  public Entity setEnemyType(EnemyType enemyType) {
+  public void setEnemyType(EnemyType enemyType) {
```

Comment:
I believe the purpose of returning entity here was to allow for chaining e.g. 
```java
new Entity()
                        .addComponent(new TextureRenderComponent(imagePath))
                        .addComponent(new CameraZoomComponent())
                        .addComponent(new PhysicsComponent(true))
                        .setEnemyType(enemyType);
```

Reply:
Fixed